applications/nrf_desktop/configuration/nrf52kbd_nrf52832/hid_keymap_def.h

```diff
@@ -6,6 +6,7 @@
 
 #include "hid_keymap.h"
 #include <caf/key_id.h>
+#include "fn_key_id.h"
```

Comment:
I don't like mixing local and and system includes :( It looks messy

Let's keep fn_key_id.h together with hid_keymap.h

Reply:
Fixed.